pkg/sql/catalog/bootstrap/metadata.go

```diff
@@ -452,6 +453,75 @@ func addSystemDatabaseToSchema(
 	addSystemDescriptorsToSchema(target)
 	addSplitIDs(target)
 	addZoneConfigKVsToSchema(target, defaultZoneConfig, defaultSystemZoneConfig)
+	addSystemTenantEntry(target)
+}
+
+// addSystemTenantEntry adds a kv pair to system.tenants to define the initial
+// system tenant entry.
+func addSystemTenantEntry(target *MetadataSchema) {
+	info := descpb.TenantInfo{
+		ID:    roachpb.SystemTenantID.ToUint64(),
+		Name:  "system",
+		State: descpb.TenantInfo_ACTIVE,
+	}
+	infoBytes, err := protoutil.Marshal(&info)
+	if err != nil {
+		panic(err)
+	}
+
+	// Find the system.tenant descriptor in the newly created catalog.
+	// TODO(knz): Is there a better way?
```

Comment:
You could call `target.ForEachCatalogDescriptor` but it's basically the same thing as this for-loop. Feel free to add a `FindTableDescriptorByName` method if you want.